xml/System.Buffers.Binary/BinaryPrimitives.xml

```diff
@@ -49,10 +49,19 @@
         <Parameter Name="source" Type="System.ReadOnlySpan&lt;System.Byte&gt;" />
       </Parameters>
       <Docs>
-        <param name="source">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="source">The read-only span to read.</param>
+        <summary>Reads an <see cref="T:System.Int16" /> out of a read-only span of bytes as big endian.</summary>
```

Comment:
I am fine with this, but I also like your previous suggestion @carlossanlop to change this to:
```suggestion
        <summary>Reads an <see cref="T:System.Int16" /> from the beginning of a read-only span of bytes, as big endian.</summary>
```

Up to you. Note the addition of the comma on the Read overloads, similar to the write ones.